components/Footer.js

```diff
@@ -17,7 +17,7 @@ export default function Footer() {
           <div className="mr-14 w-full md:w-auto">
             <div className="">
               <Link href="/">
-                <a className="cursor-pointer">
+                <a className="cursor-pointer" aria-label="AsyncAPI homepage">
```

Comment:
```suggestion
                <a className="cursor-pointer" aria-label="AsyncAPI">
```

Reply:
Ok, but `aria-label="AsyncAPI"` reads as: "AsyncAPI link". It doesn't really show that it's going to the homepage.

Reply:
`homepage` sounds bit off. We currently do not have any specific route known as `/home`. Better to use `AsyncAPI`. The interpretation will AsyncAPI Link, that itself communicates the link will open the root route of the asyncapi
@Olaleye-Blessing